config.json

```diff
@@ -525,6 +525,14 @@
       "difficulty": 1,
       "topics": null
     },
+    {
+      "slug": "affine-cipher",
+      "uuid": "d1267415-aff5-411d-b267-49a4a2c8fda2",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": null
+    },
```

Comment:
Please add new exercises to the end of the config file.

Make sure you list the topics this exercise covers.